cluster/examples/kubernetes/rook-tools.yaml

```diff
@@ -16,6 +16,11 @@ spec:
           secretKeyRef:
             name: rook-ceph-mon
             key: admin-secret
+      - name: ROOK_MON_ENDPOINTS
+        valueFrom:
+          configMapKeyRef:
+            name: rook-ceph-mon-endpoints
```

Comment:
This could make the toolbox stale. Would it be better to give it the rook service account and roles so that it can query the configmap? 

I guess that is ok since people using the toolbox would want to use it only to debug things and not having as long running pod.

Reply:
i think the toolbox has that problem already anyways, right? because the init script that looks up the mon endpoints is only called when the pod is first created.

Reply:
Shouldn't this be a `projected` `volumeMount` to the ConfigMap `rook-ceph-mon-endpoints` instead of an environment variable?
A ConfigMap volumeMount is kept up2date on disk. So the toolbox would only need to be to re-read the file on changes in the mon map and not deleted/restarted.

Reply:
yes, i'll make that change. What are your thoughts on updating the ceph.conf automatically? Maybe the toolbox.sh could stay in a loop and update it every minute. Doesn't seem worth something more complicated like a cron job